### DISCUSSION TITLE
Using Spinner (or Loading Screen) while the Application is loading | Shiny Fluent UI

### DISCUSSION THREAD

**Discussion by samiullah27** [Appsilon/shiny.fluent#144] [Q&A] [answered]
I am creating an application using shiny fluent UI. The app contains up to 8 pages each having some graphs in them. One issue that I am facing is that whenever I run the app, 1st page (which is my default home page) shows up and I can not move to another page till 30 seconds after each graph is rendered on all pages. Now I want to put a loading screen either on each page so that whenever I switch pages, it shows a loading spinner and then graphs after rendering, or on the entire application so that the app shows up after rendering all graphs, meanwhile showing Loading Spinner. 

I would appreciate any help in this regard,

Best,
Sami

**Comment by jakubsob** [comment, marked as answer]
Hi @samiullah27,

we don't fully support Fluent spinners yet. If you're interested in using it - here is an example using a workaround: https://github.com/Appsilon/shiny.react/issues/22. The
